content/docs/concepts/oracle-network/_index.md

```diff
@@ -132,13 +132,62 @@ To exemplify:
 |`Round n+1`| Oracles submit `commits` for the _new_ symbol set and `reveals` for the _old_ symbol set |
 |`Round n+2`| Oracles submit `commits` and `reveals` for the _new_ symbol set  |
 
+
+## Data adaptors - plugin architecture
```

Comment:
Do these oracle related changes need review as well?

Reply:
No, I have checked all these aginst develop. This is from rebase and restoring linear history.